config/samples/cloudaccount_aws.yaml

```diff
@@ -1,6 +1,7 @@
 # Add AWS Account and Onboard VPC
 # To get base64 encoded json string for secret credential, run:
-# echo '{"accessKeyId": "YOUR_AWS_ACCESS_KEY_ID", "accessKeySecret": "YOUR_AWS_ACCESS_KEY_SECRET", "roleArn": "YOUR_AWS_IAM_ROLE_ARN", "externalId": "IAM_ROLE_EXTERNAL_ID"}' | openssl base64
+# echo '{"accessKeyId": "YOUR_AWS_ACCESS_KEY_ID", "accessKeySecret": "YOUR_AWS_ACCESS_KEY_SECRET", "roleArn": "YOUR_AWS_IAM_ROLE_ARN",
+# "externalId": "IAM_ROLE_EXTERNAL_ID"}' | openssl base64
```

Comment:
This might cause JSON parser error as discussed.

Reply:
Done